src/components/frame/v5/pages/TeamsPage/hooks.tsx

```diff
@@ -238,5 +252,110 @@ export const useTeams = (): TeamCardListProps['items'] => {
       ];
     }, []) || [];
 
-  return teams.sort((a, b) => (a.reputation > b.reputation ? -1 : 1));
+  const [searchValue, setSearchValue] = useState('');
+
+  const [value, setValue] = useState<Partial<TeamsPageFilters>>({
+    funds: {
+      descending: true,
+      ascending: false,
+    },
+    reputation: {
+      descending: false,
+      ascending: false,
+    },
+  });
+
+  const sortTeam = (a, b) => {
+    const { funds, reputation } = value;
+
+    if (a.balanceValue !== b.balanceValue) {
+      if (funds?.ascending) {
+        return a.balanceValue - b.balanceValue;
+      }
+      if (funds?.descending) {
+        return b.balanceValue - a.balanceValue;
+      }
+    }
+
+    if (a.reputation !== b.reputation) {
+      if (reputation?.ascending) {
+        return a.reputation - b.reputation;
+      }
+      if (reputation?.descending) {
+        return b.reputation - a.reputation;
+      }
+    }
+
+    return 0;
+  };
+
+  const sortedTeams = teams.sort(sortTeam);
```

Comment:
`.sort` modifies the original array so please make sure you're using spread operator like so to avoid unexpected behaviour down the line: `[...teams].sort()`